docs/DEPLOYMENT.md

```diff
@@ -121,30 +138,17 @@ After the deployment is done successfully, we need to expose the URL to access t
 For `OpenShift`:-
 
 ```bash
-curl -k -X GET -I $(oc get routes api --template='https://{{ .spec.host }}/categories')
+curl -k -X GET -I $(oc get routes api --template='https://{{ .spec.host }}/v1/categories')
 ```
 
-### Setup Catalog Refresh Secret
-
-1. Get the Hub Token
-2. Make sure you have your Github ID in Catalog Refresh scope present in [config.yaml][config.yaml]
-3. Navigate to `config/99-post-deploy` and edit `34-catalog-refresh-secret.yaml`. Set the Hub Token
-
-   ```yaml
-   apiVersion: v1
-   kind: Secret
-   metadata:
-     name: catalog-refresh
-     namespace: tekton-hub
-   type: Opaque
-   stringData:
-     HUB_TOKEN: hub token
-   ```
+NOTE: At this moment, there are no resources in the db. Only the tags and categories from hub config are added in the db.
```

Comment:
After creating the link, can you add the link here with a line saying, for adding the resources follow the catalog refresh steps, wdyt ??

Reply:
I know we have added to deploy the ui first but just a suggestion